src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -298,12 +303,70 @@ def _start_cinema4d_client(self) -> None:
             new_module_path = plugin_dir + os.pathsep + module_path
         os.environ[module_path_key] = new_module_path
 
+        arguments = [c4d_exe, "-nogui", "-DeadlineCloudClient"]
+        if "linux" in platform.system().lower():
+            _logger.info("Setting Linux Cinema4D environment")
+            env = self._get_cinema4d_environment(c4d_exe)
+            _logger.info("Inserting Linux adaptor wrapper script")
+            arguments.insert(0, os.path.join(os.path.dirname(__file__), "adaptor.sh"))
```

Comment:
If we're using the c4d provided `setup_env` script in the `adaptor.sh`, does the python do something that's materially different? We seem to be splitting the functionality across the two parts. Should we combine this into 1? ie. just the shell script?

Reply:
Good catch. I'll move it over to the wrapper